library/std/src/net/ip.rs

```diff
@@ -118,16 +110,81 @@ pub struct Ipv6Addr {
     inner: c::in6_addr,
 }
 
-#[allow(missing_docs)]
+/// Scope of an IPv6 address as defined in [section 2 of IETF RFC 7346]
+///
+/// # Stability guarantees
+///
+/// This enum may [may by subject to changes][changes] in the
+/// future, as new RFCs are published.
+///
+/// [changes]: ../net/index.html#stability-guarantees-for-ietf-defined-behavior
+/// [section 2 of IETF RFC 7346]: https://tools.ietf.org/html/rfc7346#section-2
 #[derive(Copy, PartialEq, Eq, Clone, Hash, Debug)]
+#[non_exhaustive]
+#[stable(feature = "ip", since = "1.42.0")]
```

Comment:
`1.42.0`? Shouldn't this be `1.47.0`?
```suggestion
#[stable(feature = "ip", since = "1.47.0")]
```

Reply:
Yes! Like I said my comment, I haven't actually written any code yet, I just wanted to continue the conversation and see if this was still something that was needed / wanted and make sure that I was clear to start working on it :)